core/templates/components/state-editor/state-solution-editor/state-solution-editor.directive.html

```diff
@@ -87,6 +87,22 @@ <h3 class="oppia-exp-solution-card-header">Solution</h3>
   state-solution-editor .oppia-add-hint-button-container {
     margin: 10px 0;
   }
+
+  state-solution-editor .nav .tooltip-inner {
+    height: auto;
+    line-height: normal;
+    max-width: 20vw;
+    white-space: pre-line
+  }
+
+  state-solution-editor .tooltip-arrow {
+    left: 0.75em !important;
+  }
+
+  state-solution-editor .tooltip.bottom {
+    left: -1em !important;
+  }
+
```

Comment:
Could this be solved in a more general manner? So that if we have a tooltip in a different place than it doesn't happen there too?

Reply:
This looks like a common problem for many people. [src](https://www.google.com/search?q=Prevent+CSS+tooltip+from+going+out+of+page%2Fwindow&oq=Prevent+CSS+tooltip+from+going+out+of+page%2Fwindow&aqs=chrome..69i57j69i60.734j0j4&sourceid=chrome&ie=UTF-8)

They fixed it using a Jquery (If it is going to be a general fix).

Reply:
@gp201 Yep, you will probably need to find which library we use (if we use it) and if there is a way to fix it in there or replace it with a different library?